scripts/release/bug_bash.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, Facebook
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Query the Top-Ten Bug Bashers
+
+This script will query the top-ten Bug Bashers in a specified date window.
+
+Usage:
+    ./scripts/bug-bash.py -t ~/.ghtoken -b 2021-07-26 -e 2021-08-07
+    GITHUB_TOKEN="..." ./scripts/bug-bash.py -b 2021-07-26 -e 2021-08-07
+"""
+
+import argparse
+from datetime import datetime, timedelta
+import operator
+import os
+
+# Requires PyGithub
+from github import Github
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-a', '--all', dest='all',
+                        help='Show all bugs squashed', action='store_true', default=False)
+    parser.add_argument('-t', '--token', dest='tokenfile',
+                        help='File containing GitHub token', metavar='FILE', default=None)
+    parser.add_argument('-b', '--begin', dest='begin', help='begin date (YYYY-mm-dd)',
+                        metavar='date', type=valid_date_type, required=True)
+    parser.add_argument('-e', '--end', dest='end', help='end date (YYYY-mm-dd)',
+                        metavar='date', type=valid_date_type, required=True)
+
+    args = parser.parse_args()
+
+    if args.end < args.begin:
+        raise ValueError(
+            'end date {} is before begin date {}'.format(args.end, args.begin))
```

Comment:
Just a general note that Zephyr requires python 3.6+, so you can use f-strings if you want.